cmake/modules/AddRootBench.cmake

```diff
@@ -71,8 +71,13 @@ function(RB_ADD_GBENCHMARK benchmark)
   endif()
 
   # Add benchmark as a CTest
-  add_test(NAME rootbench-${benchmark}
-           COMMAND ${benchmark} --benchmark_out_format=csv --benchmark_out=rootbench-gbenchmark-${benchmark}.csv --benchmark_color=false)
+  if (${rootbench-out-format-json})
```

Comment:
I believe this could be more generic allowing us to add random options to pass to the gbench executable. We should not have just another cmake option per flag as that won’t scale.